common/scala/src/main/scala/whisk/connector/kafka/KafkaConsumerConnector.scala

```diff
@@ -157,6 +185,10 @@ class KafkaConsumerConnector(
           }
         }
       }
+    }.andThen {
+      case Failure(e) =>
+        // Only log level info because failed metric reporting is not critical
+        logging.info(this, s"lag metric reporting failed for topic '${topic}': ${e}")
```

Comment:
I would rather keep it on `info` for the moment because we don't know yet how many of these messages we are going to see in a large env. In the past, all problems were unnoticed. If the message should occur too often, it can be suppressed by setting log level to `warning`.